integration-cli/docker_cli_push_test.go

```diff
@@ -184,13 +151,8 @@ func (s *DockerTrustSuite) TestTrustedPushWithFaillingServer(c *check.C) {
 	pushCmd := exec.Command(dockerBinary, "push", repoName)
 	s.trustedCmdWithServer(pushCmd, "https://example.com:81/")
 	out, _, err := runCommandWithOutput(pushCmd)
-	if err == nil {
-		c.Fatalf("Missing error while running trusted push w/ no server")
-	}
-
-	if !strings.Contains(string(out), "error contacting notary server") {
-		c.Fatalf("Missing expected output on trusted push:\n%s", out)
-	}
+	c.Assert(err, check.NotNil, check.Commentf("Missing error while running trusted push w/ no server"))
+	c.Assert(out, checker.Contains, "error contacting notary server", check.Commentf("Missing expected output on trusted push:\n%s", out))
```

Comment:
And here (and below.. I think I can omit the next ones :stuck_out_tongue_closed_eyes:)